cmd/mattermost-mattermod/main.go

```diff
@@ -74,7 +74,11 @@ func main() {
 
 	c := cron.New()
 
-	_, err = c.AddFunc("0 2 * * *", s.RefreshMembers)
+	_, err = c.AddFunc("0 2 * * *", func() {
+		time.Sleep(time.Duration(5) * time.Second)
```

Comment:
We need to add the jitter part, which means `rand.Intn(6) * time.Second`. Always sleeping for 5 second means the same thing. We want to spread the load between the two services so that rate limits are not eaten up at the same time.

Reply:
Q: Why don't just run at different timstamps, using different crons for the two services? 10 minutes or 1 hour apart for example. Wouldn't that make it more safe for API rate limiting?

Reply:
I've tested code little bit more, when `jobserver` and `mattermod` has different set of users, that will cause no issue. Jobserver uses Org Members data only mark contributor PR's with stale label when conditions are met. Since there will be no issue, i am also in favor of starting mattermod cron job later ie. `10 2 * * *`.

Reply:
Configured mattermod cron job 10 minute later than jobserver.

Reply:
Yep, agree with Akis.